CHANGELOG.unreleased.md

```diff
@@ -25,6 +27,8 @@ Section headings should be at level 3 (e.g. `### Added`).
 
 This version removes the legacy implementaion of the `service` process. This is a breaking change.
 
+This version raises errors that would previously have suppressed during calls to `Artifact.link()` or `Run.link_artifact()`.  While this prevents undetected failures in those methods, it is also a breaking change.
```

Comment:
super nit: single space after a sentence instead of a double space :)


```suggestion
This version raises errors that would previously have suppressed during calls to `Artifact.link()` or `Run.link_artifact()`. While this prevents undetected failures in those methods, it is also a breaking change.
```